### PR TITLE
Added new fur-lined green trenchcoat.

### PR DESCRIPTION
**Pull request by YourFatherFigure**
Adds a new trenchcoat sprited by me. Credit to Myazaki for guiding through the process.

### CODE REVIEW

**Comment by YourFatherFigure** [comment]
!review

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -441,6 +441,15 @@
...
 	icon_closed = "trenchcoat_grey"
 
+/obj/item/clothing/suit/storage/toggle/trench/greentrench
+    name = "Green Trenchcoat"
+    desc = "A comfy looking green trenchcoat with fur."
+    icon_state = "greentrench"
+    item_state = "greentrench"
+    icon_open = "greentrench_open"
+    icon_closed = "greentrench"
+    body_parts_covered = UPPER_TORSO|ARMS
```
Comment (Alberyk) [inline code/modules/clothing/suits/miscellaneous.dm]: Those vars are redundant, can be safely removed.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -441,6 +441,15 @@
 	icon_open = "trenchcoat_grey_open"
 	icon_closed = "trenchcoat_grey"
 
+/obj/item/clothing/suit/storage/toggle/trench/greentrench
+    name = "Green Trenchcoat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
    name = "green trenchcoat"
```
Clothing and items in general should not have capitalized names.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -441,6 +441,15 @@
 	icon_open = "trenchcoat_grey_open"
 	icon_closed = "trenchcoat_grey"
 
+/obj/item/clothing/suit/storage/toggle/trench/greentrench
```
Comment (Arrow768) [inline code/modules/clothing/suits/miscellaneous.dm]: Rename it to 
```suggestion
/obj/item/clothing/suit/storage/toggle/trench/green
```
The "trench" exists already as part of the path
Dont forget to change the path of the loadout item aswwell

**Comment by Arrow768** [comment]
Same for the icon/item states.
Change them to be in line with the standard set by the other trenchcoats

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -441,6 +441,15 @@
...
 	icon_closed = "trenchcoat_grey"
 
+/obj/item/clothing/suit/storage/toggle/trench/green
+    name = "green trenchcoat"
+    desc = "A comfy looking green trenchcoat with fur."
+    icon_state = "greentrench"
+    item_state = "greentrench"
+    icon_open = "greentrench_open"
+    icon_closed = "greentrench"
+    body_parts_covered = UPPER_TORSO|ARMS
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/miscellaneous.dm]: See above, you can get rid of this variable I'm pretty sure.

**Pull request merged**